proof/courier.go

```diff
@@ -622,10 +639,17 @@ func (b *BackoffHandle) Exec(ctx context.Context, proofLocator Locator,
 		return fmt.Errorf("backoff config not specified")
 	}
 
+	locatorHash, err := proofLocator.Hash()
+	if err != nil {
+		return err
+	}
+	log.Infof("Starting proof delivery backoff procedure for proof: "+
+		"(locator_hash=%x)", locatorHash)
```

Comment:
I think this logs as double hashed, since it's a hash that has a `String()` method. So should proably be `locatorHash[:]` instead.